js/src/components/contact-information/store-address-card.scss

```diff
@@ -0,0 +1,12 @@
+.gla-store-address {
+	&__refresh-button {
+		position: absolute;
+		top: 0;
+		bottom: 0;
+		margin: auto;
```

Comment:
:nail_care: :scroll: 
I think we could, make the icon a (horizontal) center of focus area:
##### Suggested
![image](https://user-images.githubusercontent.com/17435/128050959-90f2f75a-c884-4e51-857f-90348c9d02a6.png)
##### Current
![image](https://user-images.githubusercontent.com/17435/128051055-10fa550a-5ff2-4ddf-9518-759ff00be34f.png)
```suggestion
		margin: auto;

		&.has-icon.has-text {
			justify-content: center;

			svg {
				margin: 0;
			}
		}
```

Or __preferably__, improve our `<AppButton>` implementation, to remove unnecessary whitespace text child nodes, if there are no children https://github.com/woocommerce/google-listings-and-ads/blob/trunk/js/src/components/app-button/index.js#L58-L61 so the `<button>` will not get `.has-text` class, therefore, the `left` alignment.

Reply:
Thanks! I took your suggested change first. It's more proper to improve the **AppButton** but there are a few tricky things that need much time to make sure all usages work well. I will open another PR later.